libraries/networking/src/NLPacketList.h

```diff
@@ -12,17 +12,26 @@
 #ifndef hifi_NLPacketList_h
 #define hifi_NLPacketList_h
 
+#include <list>
+
 #include "udt/PacketList.h"
 
+#include "NLPacket.h"
+
 class NLPacketList : public udt::PacketList {
 public:
-    NLPacketList(PacketType packetType, QByteArray extendedHeader = QByteArray());
+    NLPacketList(PacketType packetType, QByteArray extendedHeader = QByteArray(), bool isReliable = false, bool isOrdered = false);
+    NLPacketList(PacketList&& packetList);
+
+    QUuid getSourceID() const { return _sourceID; }
```

Comment:
Though I think COW takes care of this I still like to make these returns `const QUuid&` where possible